src/mmw/js/src/draw/views.js

```diff
@@ -19,13 +19,30 @@ var ToolbarView = Marionette.LayoutView.extend({
     template: toolbarTmpl,
 
     regions: {
-        selectAreaRegion: '#select-area-region',
+        selectTypeRegion: '#select-area-region',
         drawAreaRegion: '#draw-area-region',
         placeMarkerRegion: '#place-marker-region'
     },
 
+    modelEvents: {
+        'change:toolsEnabled': 'manageOutlineLayer'
+    },
+
+    initialize: function() {
+        var map = App.getLeafletMap();
+        var ofg = this.model.outlineFeatureGroup = L.featureGroup();
```

Comment:
Yeah, this feature group is a special case. It isn't an attribute that needs to be persisted to the model necessarily... but I guess it wouldn't hurt to make it an attribute either. I'm still thinking of a good design pattern for handling extra layers outside of the map view.